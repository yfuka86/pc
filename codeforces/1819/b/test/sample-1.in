4
3
1 2
3 5
1 3
3
1 1
1 1
1 1
1
10 10
4
3 2
5 5
2 2
8 7
