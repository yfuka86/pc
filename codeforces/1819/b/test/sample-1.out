1
4 5
2
1 3
3 1
1
10 10
1
13 7
